src/main/java/com/libertymutualgroup/herman/aws/ecs/EcsPush.java

```diff
@@ -218,7 +219,7 @@ public EcsPush(EcsPushContext context) {
 
         this.lambdaClient = AWSLambdaClientBuilder.standard()
             .withCredentials(new AWSStaticCredentialsProvider(context.getSessionCredentials()))
-            .withClientConfiguration(context.getAwsClientConfig())
+            .withClientConfiguration(new ClientConfiguration().withClientExecutionTimeout(300000).withSocketTimeout(300000))
```

Comment:
Maxing execution and socket timeout values to ensure broker lambdas don't run multiple times